features/gba-dev/gezagsverhouding-gba.feature

```diff
@@ -0,0 +1,142 @@
+#language: nl
+
+Functionaliteit: GBA Persoon: gezagsverhouding
+
+  Abstract Scenario: persoon heeft 'gezagsverhouding' veld: 'indicatie gezag minderjarige (32.10)'
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'gezagsverhouding' gegevens
+    | naam                                 | waarde |
+    | indicatie gezag minderjarige (32.10) | 12     |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000152                       |
+    | fields              | <naam>                          |
+    Dan heeft de response een persoon met de volgende gegevens
+    | naam                                    | waarde           |
+    | indicatieGezagMinderjarige.code         | 12               |
+    | indicatieGezagMinderjarige.omschrijving | ouder1 en ouder2 |
```

Comment:
dit komt niet overeen met de omschrijving in de landelijke tabellen pdf. Het zou 'Ouder1 en ouder2 hebben het gezag' moeten zijn

Reply:
Aangepast